v21.1/add-region.md

```diff
@@ -6,12 +6,14 @@ toc: true
 
 <span class="version-tag">New in v21.1:</span> The `ALTER DATABASE .. ADD REGION` [statement](sql-statements.html) adds a [region](multiregion-overview.html#database-regions) to a [multi-region database](multiregion-overview.html).
 
+{% include enterprise-feature.md %}
```

Comment:
Did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}" ?